### DISCUSSION TITLE
CVS noticesubtype 4 problem - 403

### DISCUSSION THREAD

**Discussion by elerik** [OP-TED/eForms-SDK#950] [Q&A] [answered]
Hi Labs, 

I have just noticed that when i use csv server to validate subnotice 4 i have answer 403. When i use notice subtype 16 everything is ok. 29 everything is ok, 7, 10 ok. And so on. But 4 return 403. 
At csv preview it works properly. 
Can you please check if everything is correctly set at live cvs server specificaly for notice sub type 4 ? 
Thanks and best regards.

(SDK 1.11)

**Comment by YvesJo** [comment, marked as answer]
Please get in touch directly with the support for this non SDK related issue.

**Reply by elerik** [reply]
Iam going to place answer to all if anybody will search for this.

> 1/ The request body is less than 8192 bytes
2/ The request does not have a "User-Agent" header

> The error does not originate from the CVS application, but from another component that is in front of it (firewall/reverse-proxy).

> It is not specific to the notice subtype 4, but for the other notice subtypes the XML is usually larger. It is probably the case for your other notices.

> We recommend that you set a "User-Agent" header in your requests to our systems (most tools and libraries do it by default). The value itself does not matter, it's just that the absence of this header can be sometimes considered as a sign of a suspicious request.

**Reply by YvesJo** [reply]
Thanks for sharing, I'll reopen the ticket so that the probability for people to detect it will increase.
